wolox-training/rof-angular
# Implemented view for user's rents

## Summary

Implemented view for user's rents

## Test Plan

* Log into your account
* Go to 'My Rents'
* Check your rents

## Trello Card 

 https://trello.com/c/ifC3Jl4F

## Screenshot

![my_rents](https://cloud.githubusercontent.com/assets/2911971/10833728/edce6bd2-7e72-11e5-92f3-9524ced4cd9b.png)
